runtime/jcl/common/reflecthelp.c

```diff
@@ -837,7 +837,16 @@ createField(struct J9VMThread *vmThread, jfieldID fieldID)
 			|| J9ROMCLASS_IS_RECORD(j9FieldID->declaringClass->romClass)
 			|| J9ROMCLASS_IS_HIDDEN(j9FieldID->declaringClass->romClass)
 		) {
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+			U_32 flags = 0x10; /* java.lang.Field:TRUST_FINAL */
```

Comment:
Instead of using the value directly here, please define macro `TRUST_FINAL` and `TRUST_FINAL`